applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -594,7 +623,10 @@ Use the following UART configuration:
 
 .. note::
    The GPIO output level on the nRF91 Series device side must be 3 V.
-   You can set the VDD voltage with the **VDD IO** switch (**SW9**).
+
+   For nRF91x1 DK, you can set the VDD voltage with the `Board Configurator app`_.
+
+   For nRF9160 DK, you can set the VDD voltage with the **VDD IO** switch (**SW9**).
```

Comment:
```suggestion
   For an nRF91x1 DK, you can set the VDD voltage using the `Board Configurator app`_.
   For an nRF9160 DK, you can set the VDD voltage with the **VDD IO** switch (**SW9**).
```
You might even consider having these two as bullet points.

Reply:
Changed to bullets. Let's see how it generates in a note.